parallel/20200722/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="GNU Parallel"
+LABEL software.version="20200722"
+LABEL container.version="1"
```

Comment:
Please delete the container.version LABEL since that info is captured (more coherently) with dockerfile.version